Plans and more

upayanmazumder opened the pull request.

upayanmazumder committed:
🛠️ global.css -> Added light and dark red colors.
🛠️ index.tsx -> Added Plans component, modified ellipsis color.
🛠️ styles.css -> Changed colors and added box-shadow to buttons, modified gradients and colors for button-small and ellipsis.

upayanmazumder committed:
Added dynamic routing for pages!

upayanmazumder committed:
🟢 plans.json - New file for hosting plans, AI, game, VPS, VDS, bare-metal, development services, PC build options
🟢 index.tsx - New file for rendering plans and setting document head metadata

upayanmazumder committed:
🛠️ mcplans.tsx -> Updated plansData import
🛠️ vpsplans.tsx -> Updated plansData import
🟢 introplans.json for new file
🔴 plans.json

upayanmazumder committed:
Added breadcrumb

upayanmazumder committed:
Updates

upayanmazumder committed:
Updates

upayanmazumder committed:
🔴 qwik/src/components/plans/mcplans/mcplans.tsx
🔴 qwik/src/components/plans/plans.tsx
🟢 qwik/src/components/introplans/planstyles.module.css
🛠️ qwik/src/components/plans/vpsplans/vpsplans.tsx -> Renamed to qwik/src/components/introplans/vpsplans/vpsplans.tsx
🟢 qwik/src/components/plans/hostingplans/hostingplans.tsx
🛠️ qwik/src/components/plans/planstyles.module.css -> Updated styles for plans
🛠️ qwik/src/data/plans.json -> Updated plan descriptions and added links
🛠️ qwik/src/routes/index.tsx -> Updated import path for Plans component
🛠️ qwik/src/routes/plans/hosting/index.tsx -> Added HostingPlans component

upayanmazumder committed:
Nav bar now better

upayanmazumder committed:
🛠️ hostingplans.tsx -> Updated category styles and added links to plans
🛠️ planstyles.module.css -> Adjusted background colors and added margin
🛠️ plans.json -> Renamed categories to uppercase, added links to plans

upayanmazumder committed:
🔴 .gitignore
🟢 .gitignore

upayanmazumder committed:
🟢 qwik/src/components/plans/hostingplans/dynamic.tsx
🛠️ qwik/src/components/plans/hostingplans/hostingplans.tsx -> Updated import path
🔴 qwik/src/components/plans/planstyles.module.css
🛠️ qwik/src/data/nav.json -> Updated route for Website
🛠️ qwik/src/routes/plans/hosting/[hostId]/index.tsx -> Added import for dynamic plans
🛠️ qwik/src/routes/plans/hosting/index.tsx -> Removed unnecessary comment

upayanmazumder committed:
🛠️ plans.module.css -> Removed display: flex, Added vertical-align, text-align, align-items, justify-content in .category, Added comments for more customization, Added transition in .price

upayanmazumder committed:
fix ig

upayanmazumder committed:
Added maps

upayanmazumder committed:
🛠️ index.tsx -> Changed width of map from 100% to 80%
🛠️ index.tsx -> Replaced a div with id="map" inside .container-center

upayanmazumder committed:
Merge branch 'main' of https://github.com/Eranodes/eranodes.com into Plans-and-more

Xtimate committed:
Styled Hosting page

Xtimate committed:
Fixed transparency

Xtimate reviewed (approved):
Good job! You're doing alot more then me.

Xtimate committed:
Added Tailwind

Xtimate committed:
Revert "Added Tailwind"
Reverted TailwindCSS
This reverts commit 1f0c3ea49bb99163cfaaaa593780e0cfe29a3ef8.

Xtimate committed:
Made circle diagram and counting animation

upayanmazumder committed:
gg

upayanmazumder committed:
🛠️ circlediagram.module.css -> Updated circle dimensions. Added text-wrap property to labels.
🛠️ circlediagram.tsx -> Removed label prop. Refactored code structure.
🛠️ statistics.module.css -> Restructured CSS classes for better layout.
🛠️ statistics.tsx -> Restructured component layout for statistics display. Updated label elements.

upayanmazumder committed:
Merge branch 'main' into Plans-and-more

upayanmazumder committed:
🔴 qwik/src/components/introplans/mcplans/mcplans.tsx
🔴 qwik/src/components/introplans/plans.tsx
🔴 qwik/src/components/introplans/planstyles.module.css
🔴 qwik/src/components/introplans/vpsplans/vpsplans.tsx
🛠️ qwik/src/components/plans/hostingplans/dynamic.tsx -> Updated dynamic hosting plans
🛠️ qwik/src/components/plans/hostingplans/hostingplans.tsx -> Updated hosting plans component
🔴 qwik/src/components/plans/planstyles.module.css
🟢 qwik/src/data/nav.json for new navigation data
🔴 qwik/src/routes/demo/flower/flower.css
🔴 qwik/src/routes/demo/flower/index.tsx
🔴 qwik/src/routes/demo/todolist/index.tsx
🔴 qwik/src/routes/demo/todolist/todolist.module.css
🔴 qwik/src/routes/index.tsx
🟢 qwik/src/routes/plans/hosting/[hostId]/index.tsx for new hosting plans route
🟢 qwik/src/routes/plans/hosting/index.tsx for new hosting plans route
🔴 qwik/src/routes/plans/pc-build/[typeId]/index.tsx
🔴 qwik/src/routes/plans/pc-build/index.tsx

upayanmazumder reviewed (commented):
Updates

upayanmazumder committed:
🛠️ qwik/src/components/breadcrumb/breadcrumb.module.css -> Adjusted padding and margin
🛠️ qwik/src/components/breadcrumb/breadcrumb.tsx -> Added getBreadcrumb function to compute breadcrumb
🛠️ qwik/src/components/header/header.module.css -> Modified height and background color
🟢 qwik/src/data/introplans.json for new files
🔴 qwik/src/routes/layout.tsx for deleted files

upayanmazumder committed:
Will work on this later

upayanmazumder committed:
🔴 .gitignore
🛠️ qwik/src/components/calculator/vps/vps.tsx -> added eslint-disable line.

upayanmazumder committed:
🔴 qwik/src/data/nav.json
🔴 qwik/src/routes/plans/hosting/[hostId]/index.tsx
🟢 qwik/src/routes/plans/hosting/[hostId]/hostingplans.module.css
The changes include the deletion of nav.json and index.tsx files, and the addition of a new file hostingplans.module.css. Additionally, there are modifications in the nav.json file to update the route for the Website option, and in the index.tsx file to incorporate dynamic loading of hosting plans based on the selected hostId.

upayanmazumder committed:
🔴 dynamic.tsx
🔴 planstyles.module.css
🔴 hostingplans.module.css
🛠️ index.tsx -> Updated styles import
🛠️ index.tsx -> Updated component class and structure

upayanmazumder committed:
🛠️ plans.module.css -> Added new styles for .title
🛠️ index.tsx -> Added hosting icons, formatted hostId, modified UI to display hosting plans for different categories.

upayanmazumder committed:
More Modular

upayanmazumder committed:
fix spacing
